### PR TITLE
Add variants of insert, insensitive, instruct, sensitive

### PR DESCRIPTION
**Pull request by bl-ue**
Hello again!

I discovered these myself, in my own code, and found that codespell didn't know about them.

### CODE REVIEW

**Comment by bl-ue** [comment]
ping @peternewman.  I didn't realize this whole time that the CI had failed 😆

**Comment by peternewman** [comment]
> ping @peternewman. I didn't realize this whole time that the CI had failed 😆

It hasn't, or not in a way that matters. I just haven't personally had time to review it.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16583,6 +16592,12 @@ intrumenting->instrumenting
 intruments->instruments
 intrusted->entrusted
 intstead->instead
+intstruct->instruct
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:16595]: Definitely suggest `in struct` possibly suggest `int struct` if that can appear in some languages.

**Review by peternewman — CHANGES_REQUESTED** [review]
A comment

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (bl-ue) [reply codespell_lib/data/dictionary.txt]: → 614ad8af3ef22c43bd64041dc2d0b1d2af3e99a4

Comment (bl-ue) [reply codespell_lib/data/dictionary.txt]: I didn't add `int struct` because in C, C++ and some other languages, `struct` is a keyword.

**Comment by bl-ue** [comment]
ping @peternewman :)

**Pull request merged**